src/app/public/modules/popover/popover.component.ts

```diff
@@ -166,7 +169,10 @@ export class SkyPopoverComponent implements OnInit, OnDestroy {
 
     // Let the styles render before gauging the dimensions.
     this.windowRef.getWindow().setTimeout(() => {
-      if (this.adapterService.isPopoverLargerThanParent(this.popoverContainer)) {
+      if (
+        this.allowFullscreen
+        && this.adapterService.isPopoverLargerThanParent(this.popoverContainer)
```

Comment:
Just for consistency's sake, mind putting the `&&` at the end of the first line? Same for below.

Reply:
Done.